bin/tachyon-mount.sh

```diff
@@ -213,6 +213,7 @@ case "${1}" in
         mount_local $1
         ;;
       workers)
+        bin=`cd "$( dirname "$0" )"; pwd`
         $LAUNCHER $bin/tachyon-workers.sh $bin/tachyon-mount.sh $1
```

Comment:
I would do that in a separate PR. It would be great to apply the Google shell style (https://google.github.io/styleguide/shell.xml) as a follow if that is something that interests you.